libs/subliminal_patch/score.py

```diff
@@ -4,6 +4,7 @@
 
 from subliminal.video import Episode, Movie
 from subliminal.score import get_scores
+from config import settings
```

Comment:
I don't see any reason to modify this file. Could you update your PR to exclude this file? Thanks.

Reply:
After that I'll be able to merge your PR to dev branch.

Reply:
My bad, these were remnants of the addic7ed_boost. I just removed it.